classloader/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.smallrye.common</groupId>
+        <artifactId>smallrye-common-parent</artifactId>
+        <version>1.1.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>smallrye-common-classloader</artifactId>
```

Comment:
WDYT about calling this `smallrye-common-lang` since it deals with `java.lang.*` classes?

Reply:
So far (dating back to WildFly Common, this project's predecessor) the naming convention has been based on the functionality of the classes, not on any particular correspondence to a JDK package.  I think this is less confusing, particularly when some functionality may abstractly cover more than one JDK package.